tls/s2n_cipher_suites.c

```diff
@@ -22,21 +22,12 @@
 
 #include "tls/s2n_cipher_preferences.h"
 #include "tls/s2n_cipher_suites.h"
-#include "tls/s2n_connection.h"
+#include "tls/s2n_client_key_exchange.h"
```

Comment:
You don't seem to be referencing any of the client or server key exchange functions from s2n_cipher_suites. Isn't including s2n_kex.h sufficient?